spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -34,23 +34,23 @@
       scenario 'user leaves phone blank' do
         sign_in_before_2fa
         fill_in 'Phone', with: ''
-        click_button t('devise.two_factor_authentication.buttons.confirm_with_sms')
+        click_button 'Submit'
 
         expect(page).to have_content invalid_phone_message
       end
 
       scenario 'user enters an invalid number with no digits' do
         sign_in_before_2fa
         fill_in 'Phone', with: 'five one zero five five five four three two one'
-        click_button t('devise.two_factor_authentication.buttons.confirm_with_sms')
+        click_button 'Submit'
 
         expect(page).to have_content invalid_phone_message
       end
 
       scenario 'user enters a valid number' do
         user = sign_in_before_2fa
         fill_in 'Phone', with: '555-555-1212'
-        click_button t('devise.two_factor_authentication.buttons.confirm_with_sms')
+        click_button 'Submit'
```

Comment:
We're going to need to be more diligent about using localized strings in favor of hardcoded values. Can we exchange this 'Submit' for a localized button value? 

This will permit updates to copy without affecting specs.